processors/gtfs.py

```diff
@@ -378,5 +369,12 @@ def make_gtfs(filename, gtfs_data):
             with io.StringIO(newline="") as string_io:
                 writer = csv.writer(string_io, delimiter=",")
                 writer.writerow(columns)
-                writer.writerows(gtfs_data[gtfs_feature])
+                writer.writerows(
+                    map(
+                        functools.partial(
```

Comment:
What's the purpose of partial here?

Reply:
The purpose is to bind the second parameter. I see the alternative
```python
                    map(
                        lambda feature_item: dict_to_row(
                            feature_item, gtfs_feature
                        ),
                        gtfs_data[gtfs_feature]
                    )
```
Would it be better this way?

Reply:
I prefer the variant with `partial`. The question is where we should declare it: inside or outside `map`?

Reply:
> I mean, it seems to me, that you're creating a new partial for each map iteration, aren't you?

We have 
```python
map(<expression that returns a partial object>, <iterable the partial is applied to>)
```
Parameter evaluation is performed before function call, I don't think the `map` function behaves differently, so only one partial object is created wherever we define them - before the map() call or inside it.

Reply:
I see, you're right!